.github/workflows/test.yml

```diff
@@ -9,6 +9,23 @@ on:
       - '.gitignore'
 
 jobs:
+  tests:
```

Comment:
🚀 Great addition of a test job! This will help ensure code quality through automated testing.